constants.py

```diff
@@ -100,7 +100,11 @@
               "bucket": "build"}
 
 VIEWS = [SERVER_VIEW, BUILD_VIEW]
-BUILDER_URLS = ["http://server.jenkins.couchbase.com/job/watson-build/"]
+
+BUILDER_URLS = ["http://server.jenkins.couchbase.com/job/couchbase-server-build/",
+                "http://server.jenkins.couchbase.com/job/watson-build/"]
+
+CHANGE_LOG_URL = "http://172.23.123.43:8282/changelog"
```

Comment:
No, in general not, but this is the constants.py anyway. It seems to me the right place to put hardcoded values/constants.